wallet/src/main/scala/org/constellation/wallet/Utils.scala

```diff
@@ -100,6 +102,32 @@ object KryoSerializer {
     kryoPool.fromBytes(bytes).asInstanceOf[T]
 }
 
+sealed trait EncodableValue// {val value: Any}
```

Comment:
`extends AnyVal`?

Reply:
It doesn't pass with extends AnyVal - illegal inheritance from a value class. The idea here was to enforce creating subclasses with a value field of arbitrary type. But `Any` looks not that pretty so it's commented out - but I think I will uncomment it to have some suggestion on what field we want.

Reply:
so maybe a generic value will work?

Reply:
Yeah we'll want a generic for serialization eventually